src/funcs/transformTableStructureToDBML.js

```diff
@@ -84,11 +80,19 @@ const getColumnDefinition = (col) => {
   return `\t"${columnName}" ${dataType}${characterMaxLength} ${columnSettings} `;
 }
 
-module.exports = function transformTableStructureToDBML({ tableName, primaryKeys, structure: colDefs }, schemaName, includeSchemaName) {
+const transformTableStructureToDBML = ({ tableName, primaryKeys, structure: colDefs }, schemaName, includeSchemaName) => {
   const columns = colDefs && Array.isArray(colDefs) ? colDefs : [];
   const columnDefinitions = columns.map(column => getColumnDefinition(column, primaryKeys));
   const tableNameString = includeSchemaName ? `${schemaName}.${tableName}` : tableName;
   columnDefinitions.unshift(`Table "${tableNameString}" {`);
   columnDefinitions.push(`} ${EOL} ${EOL} `);
   return columnDefinitions.join(`${EOL} `);
+}
+
+module.exports = {
+  transformTableStructureToDBML,
```

Comment:
A-Z ordering?

Reply:
In HPAPI-342, these have all been split out to their own files so they will have their own spec files.